src/app/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.html

```diff
@@ -1,16 +1,16 @@
 <div class="container mb-3">
-  <h1 class="form__headline">Деталі вивезення</h1>
+  <h1 class="form__headline">{{ "export-details.title" | translate }}</h1>
   <div class="form-row">
     <div class="form-group col-md-2">
-      <label>Дата вивезення</label>
+      <label>{{ "export-details.export-date" | translate }}</label>
       <input type="date" class="form-control" />
     </div>
     <div class="form-group col-md-2 mx-4">
-      <label>Час вивезення</label>
+      <label>{{ "export-details.export-time" | translate }}</label>
       <input type="time" class="form-control" />
     </div>
     <div class="form-group col-md-4">
-      <label>Станція приймання</label>
+      <label>{{ "export-details.station" | translate }}</label>
```

Comment:
add 'for' attribute to label and 'id' to inputs

Reply:
Added